docs/operation/backup.md

```diff
@@ -0,0 +1,73 @@
+# Backup Data Using BanyanDB Backup Tool
```

Comment:
This doc requires a menu item to show up in the website.

Reply:
Done